project/BuildSettings.scala

```diff
@@ -60,4 +65,10 @@ object BuildSettings {
         </developer>
       </developers>)
   )
+
+  lazy val docSettings = Seq(
+    addMappingsToSiteDir(mappings in (ScalaUnidoc, packageDoc), siteSubdirName in ScalaUnidoc),
+    gitRemoteRepo := "https://github.com/snowplow/scala-maxmind-iplookups.git",
+    siteSubdirName := "."
```

Comment:
where does `"."` publish to? aren't we fine with `""`?

Reply:
They both publish to the toplevel directory. I can change it to `""`, I didn't think to try an empty string when I committed.

Reply:
ok then let's just do `""` :+1: